### ISSUE TITLE
Undetected python error

### ISSUE THREAD

**Issue by kaihowl** [kaihowl/dotfiles#441]
```
2022-08-27T14:12:09.4256206Z ERROR: autopep8 1.7.0 has requirement pycodestyle>=2.9.1, but you'll have pycodestyle 2.8.0 which is incompatible.
```

- [x] Why is this not listed in the CI output?
- [x] Why is it there in the first place?

**Comment by kaihowl** [comment]
Not listed in CI output as we only did partial case-insensitive matching.

**Comment by kaihowl** [comment]
Fixed with #444
